### ISSUE TITLE
Opening the web admin app does not show beacons on the map 

### ISSUE THREAD

**Issue by Piiit** [noi-techpark/it.bz.beacon.api#63]
The browser reveals: error on /v1/admin/beacons

```
XHRGEThttps://api.beacon.testingmachine.eu/v1/admin/beacons
[HTTP/1.1 500  131ms]

    	
    timestamp	1606404572813
    status	500
    error	"Internal Server Error"
    message	"Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid; nested exception is org.hibernate.PropertyAccessException: Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid"
    path	"/v1/admin/beacons"

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://api.beacon.testingmachine.eu/v1/admin/beacons. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).

```

CORS is set to `*`, but it seems that the headers are not build.

**Comment by Piiit** [comment]
@sseppi FYI

**Comment by davidebz** [comment]
Can you please send me per e-mail an account on 

https://admin.beacon.testingmachine.eu

to test the problem?

**Comment by Piiit** [comment]
@davidebz You should have one already... user dmontesin

**Comment by davidebz** [comment]
The main problem here is that the API return an error 500. I will fix this first. 

@noctho can you please check this?

message: Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid; nested exception is org.hibernate.PropertyAccessException: Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid

**Comment by noctho** [comment]
> The main problem here is that the API return an error 500. I will fix this first.
> 
> @noctho can you please check this?
> 
> message: Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid; nested exception is org.hibernate.PropertyAccessException: Null value was assigned to a property [class it.bz.beacon.api.db.model.Beacon.eddystoneEid] of primitive type setter of it.bz.beacon.api.db.model.Beacon.eddystoneEid

I have fixed this with PR #66

**Comment by Piiit** [comment]
@noctho Please move the issues on your Kanban... I have merged the PR already, and it seems to we would be ready for a "Review"... Thank you
